src/core/componentSystems/MRPlaneManager.js

```diff
@@ -0,0 +1,122 @@
+import { mrjsUtils } from 'mrjs';
+import { MRPlane } from '../../datatypes/MRPlane';
+
+/**
+ * @class MRPlaneManager
+ * @classdesc creates and manages the mr.js representation of XR planes.
+ * The resulting planes have RAPIER rigid bodies and THREE.js meshes that occlude virtual content by default
+ */
+export class MRPlaneManager {
```

Comment:
if this is a manager and not a system we need to figure somewhere else for this to go - to default back to keeping inline with the ECS setup we're going for: one option to actually just change this into a System and just have an empty update function (like how the `MaskingSystem` is setup)

Reply:
Ah yes I was thinking that but forgot to move it. I'd prefer to only use systems when they're actually taking advantage of the architecture, otherwise it's a wasted update call.